ocs_ci/ocs/resources/cloud_manager.py

```diff
@@ -83,10 +95,14 @@ class S3Client(CloudClient):
     Implementation of a S3 Client using the S3 API
 
     """
-
     def __init__(self, key_id=None, access_key=None, endpoint="https://s3.amazonaws.com",
-                 verify=True, *args, **kwargs):
+                 verify=True, auth_dict=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        if auth_dict:
+            key_id = auth_dict.get('AWS_ACCESS_KEY_ID')
```

Comment:
see https://github.com/red-hat-storage/ocs-ci/pull/2543#discussion_r458665044

Reply:
I believe there isn't any alternative default value that'd be better than `None` in this case.

Reply:
ok